### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by modzelewski**
Here comes the Polish translation...
I'm a native speaker. However, it would be great to have someone else take a look and verify.

### CODE REVIEW

File: lang/pl/index.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+inkrementują wersję major.
+
+System ten został nazwany przeze mnie "Semantycznym Wersjonowaniem". W systemie tym numery wersji i sposób w jaki zostają zmienione wskazują na tworzący je kod oraz na to, co zostało zmodyfikowane od ostatniej wersji.
+
+
+Specyfikacja Semantycznego Wersjonowania (SemVer)
+------------------------------------------
+
+Następujące terminy "MUSI" ("MUST"), "NIE MOŻE" ("MUST NOT"), "WYMAGANY" ("REQUIRED"), "MA BYĆ" ("SHALL"), "NIE BĘDZIE" ("SHALL NOT"), "POWINIEN" ("SHOULD"), "NIE POWINIEN" ("SHOULD NOT"), "ZALECANY" ("RECOMMENDED"), "MOŻE" ("MAY") oraz "OPCJONALNY" ("OPTIONAL") pojawiające się w ty m dokumencie rozumiane są zgodnie z ich opisem na stronie: [RFC 2119](http://tools.ietf.org/html/rfc2119).
```
Comment (szymonwlochowski) [inline lang/pl/index.md]: `w ty m` --> `w tym`

File: lang/pl/index.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+1. MAJOR wersji, gdy dokonujesz zmian niekompatybilnych z API,
+1. MINOR wersji, gdy dodajesz nową funkcjonalność, która jest kompatybilna z poprzednimi wersjami,
+1. PATCH wersji, gdy dokonujesz naprawy błędu, która jest kompatybilna z poprzednimi wersjami. 
+
+Dodatkowe oznaczenia dla wydania przedpremierowego lub meta danych build-u są dostępne jako rozszerzenia formatu MAJOR.MINOR.PATCH.
+
+Wprowadzenie
+------------
+
+W świecie zarządzania oprogramowaniem istnieje przerażające miejsce nazywane potocznie "piekłem zależności." Im większy jest system i im więcej paczek
```
Comment (szymonwlochowski) [inline lang/pl/index.md]: "piekłem zależności." --> "piekłem zależności".

File: lang/pl/spec/v2.0.0.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+1. MAJOR wersji, gdy dokonujesz zmian niekompatybilnych z API,
+1. MINOR wersji, gdy dodajesz nową funkcjonalność, która jest kompatybilna z poprzednimi wersjami,
+1. PATCH wersji, gdy dokonujesz naprawy błędu, która jest kompatybilna z poprzednimi wersjami. 
+
+Dodatkowe oznaczenia dla wydania przedpremierowego lub meta danych build-u są dostępne jako rozszerzenia formatu MAJOR.MINOR.PATCH.
+
+Wprowadzenie
+------------
+
+W świecie zarządzania oprogramowaniem istnieje przerażające miejsce nazywane potocznie "piekłem zależności." Im większy jest system i im więcej paczek
```
Comment (szymonwlochowski) [inline lang/pl/spec/v2.0.0.md]: "piekłem zależności." --> "piekłem zależności".

**Comment by szymonwlochowski** [comment]
Very well done, just have a look at these two typos.

**Comment by modzelewski** [comment]
Thanks @szymonwlochowski !

**Comment by haacked** [comment]
I fixed this branch in #145 Thanks!

**Pull request closed without merging**